src/main/_docs/setup/setup-getting-started.md

```diff
@@ -55,22 +55,22 @@ INFO: (che start): Use: http://<your-ip>:8080
 INFO: (che start): API: http://<your-ip>:8080/swagger
 
 # Stop Che
-docker run <docker-goodness> eclipse/che-cli stop
+docker run <docker-goodness> eclipse/che stop
 
 # Restart Che
-docker run <docker-goodness> eclipse/che-cli restart
+docker run <docker-goodness> eclipse/che restart
 
 # Run a specific version of Che
-docker run <docker-goodness> eclipse/che-cli:<version> start
+docker run <docker-goodness> eclipse/che:<version> start
 
 # Get help
-docker run eclipse/che-cli
+docker run eclipse/che
 
 # If boot2docker on Windows (rare), mount a subdir of `%userprofile%` to `:/data`. For example:
```

Comment:
Let's remove `(rare)`. Anyone using virtual machine on windows will be running boot2docker so I don't know that this is rare.